### PR TITLE
security compatibility: update security compatibility with MySQL

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
updated security compatibility with MySQL in "security compatibility with MySQL" and "system variable".

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: [#5921](https://github.com/pingcap/docs/pull/5921)
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@en-jin19 Is this translation finished? In https://github.com/pingcap/docs/pull/5921/files, two docs are updated.

**Review by tiancaiamao — APPROVED** [review]

File: system-variables.md [hunk 1]
```diff
@@ -126,6 +126,13 @@ mysql> SELECT * FROM t1;
 - 这个变量表示数据存储的位置，位置可以是本地路径。如果数据存储在 TiKV 上，则可以是指向 PD 服务器的路径。
 - 如果变量值的格式为 `ip_address:port`，表示 TiDB 在启动时连接到的 PD 服务器。
 
+### default_authentication_plugin
```
Comment (TomShawn) [inline system-variables.md:129]:
```suggestion
### `default_authentication_plugin`
```

File: security-compatibility-with-mysql.md [hunk 2]
```diff
@@ -7,9 +7,29 @@ aliases: ['/docs-cn/dev/security-compatibility-with-mysql/','/docs-cn/dev/refere
...
 - 不支持列级别权限设置。
 - 不支持密码过期，最后一次密码变更记录以及密码生存期。[#9709](https://github.com/pingcap/tidb/issues/9709)
 - 不支持权限属性 `max_questions`，`max_updated`，`max_connections` 以及 `max_user_connections`。
 - 不支持密码验证。[#9741](https://github.com/pingcap/tidb/issues/9741)
+
+## 可用的身份验证插件
+
+TiDB 支持多种身份验证方式。通过使用 [`CREATE USER`](/sql-statements/sql-statement-create-user.md) 语句和 [`ALTER USER`](/sql-statements/sql-statement-create-user.md) 语句，即可创建新用户或更改 TiDB 权限系统内的已有用户。TiDB 身份验证方式与 MySQL 兼容，其名称与 MySQL 保持一致。
+
+TiDB 目前支持的身份验证方式可在以下的表格中查找到。当把身份验证方式设定为默认方法时，服务端会显示当前服务端与客户端之间的连接状态。如需进行其操作，请设置 [`default_authentication_plugin`](/system-variables.md#default_authentication_plugin) 参数。
```
Comment (TomShawn) [inline security-compatibility-with-mysql.md:19]:
```suggestion
TiDB 目前支持的身份验证方式可在以下的表格中查找到。服务器和客户端建立连接时，如要指定服务器对外通告的默认验证方式，可通过 [`default_authentication_plugin`](/system-variables.md#default_authentication_plugin) 变量进行设置。
```

File: system-variables.md [hunk 3]
```diff
@@ -126,6 +126,13 @@ mysql> SELECT * FROM t1;
...
+
+- 作用域：GLOBAL
+- 默认值：`mysql_native_password`
+- 这个变量用于设置身份验证方式。当把身份验证方式设定为默认方法时，服务端会显示当前服务端与客户端之间的连接状态。如需查询该变量的其他可选值，请参见[可用的身份验证插件](/security-compatibility-with-mysql.md#可用的身份验证插件)。
```
Comment (TomShawn) [inline system-variables.md:133]:
```suggestion
- 服务器和客户端建立连接时。这个变量用于设置服务器对外通告的默认身份验证方式。如要了解该变量的其他可选值，参见[可用的身份验证插件](/security-compatibility-with-mysql.md#可用的身份验证插件)。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**